build-infer.sh

```diff
@@ -105,7 +105,7 @@ check_installed opam
 opam init --no-setup --yes > /dev/null || \
   opam init --no-setup --yes --comp=4.02.3
 
-eval $(opam config env)
+eval $(opam config env --shell=bash)
```

Comment:
Unfortunately this doesn't work on opam 1.2.0 for instance. It seems this option was added between 1.2.0 and 1.2.2. I think this would achieve the same result:

```bash
eval $(SHELL=bash opam config env) 
```